src/Microsoft.ApplicationInsights.AspNet/Fakes.cs

```diff
@@ -1,6 +1,7 @@
 ﻿#if fakeportable
```

Comment:
Notice the amount of changes is this file was large compared to the changes in product and test code. I hope we can drop this soon.